atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -398,11 +394,11 @@ private static LockAndTimestampServices createRawEmbeddedServices(
         return AtlasDbHttpClients.createProxyWithFailover(sslSocketFactory, uris, serviceClass, userAgent);
     }
 
-    private static final class ServiceCreator<T> implements Function<ServerListConfig, T> {
+    public static final class ServiceCreator<T> implements Function<ServerListConfig, T> {
```

Comment:
Indeed, that was what I was going for, but it's not a high lift to extract. I've moved it out.